fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java

```diff
@@ -150,35 +147,54 @@ public AuthorizeController(
   }
 
```

Comment:
I realize I should not modify the function below, and create a special one to handle `CREATE`, because the function below is name-based, while for `CREATE` I want to handle the whole object not just its name.
Nevertheless, I will keep this for now for you to look at, and will modify later.
It is also worth it to (in a different PR), make a breaking change that migrates all the usages to use the full resource instead of just the name.